Alignment/OfflineValidation/macros/FitPVResiduals.C

```diff
@@ -43,13 +43,148 @@
 #include "Alignment/OfflineValidation/plugins/TkAlStyle.cc" 
 #include "CMS_lumi.C"
 
+
+/*
+  This is an auxilliary struct used to handle the plot limits
+ */
+
+struct Limits {
+
+  // initializers list
+
+  Limits() : _m_dxyPhiMax(40.),     _m_dzPhiMax(40.),       
+	     _m_dxyEtaMax(40.),     _m_dzEtaMax(40.),       
+	     _m_dxyPhiNormMax(0.5), _m_dzPhiNormMax(0.5),   
+	     _m_dxyEtaNormMax(0.5), _m_dzEtaNormMax(0.5),   
+	     _w_dxyPhiMax(120.),    _w_dzPhiMax(180.),      
+	     _w_dxyEtaMax(120.),    _w_dzEtaMax(1000.),       
+	     _w_dxyPhiNormMax(1.8), _w_dzPhiNormMax(1.8),   
+	     _w_dxyEtaNormMax(1.8), _w_dzEtaNormMax(1.8) {}   
+
+  // getter methods
+
+  std::pair <float,float> get_dxyPhiMax() const {
+    std::pair <float , float> res(_m_dxyPhiMax,_w_dxyPhiMax);
+    return res;
```

Comment:
@mmusich just a minor comment: you could save some typing if you replace these two lines by
```
return std::make_pair(_m_dxyPhiMax,_w_dxyPhiMax);